sql/updates/world/4.3.4/2021_05_01_00_world

```diff
@@ -0,0 +1,13 @@
+-- https://github.com/The-Cataclysm-Preservation-Project/TrinityCore/issues/236
+DELETE FROM `gossip_menu_option` WHERE `MenuId`=1291;
+INSERT INTO `gossip_menu_option` (`MenuId`, `OptionIndex`, `OptionIcon`, `OptionText`, `OptionBroadcastTextId`, `OptionType`, `OptionNpcflag`, `VerifiedBuild`) VALUES 
+(1291, 0, 0, 'What can I do at an inn?', 4308, 1, 1, 15595),
+(1291, 1, 5, 'Make this inn your home.', 2822, 8, 65536, 15595),
+(1291, 2, 1, 'I want to browse your goods.', 3370, 3, 128, 15595),
+(1291, 3, 0, 'Trick or Treat!', 10693, 1, 1, 15595); -- add gossip of event Hallow's End
+
```

Comment:
what sniff did you use to get that 4th gossip option? if it has a verified built it must be validated by a sniff.

Reply:
that gossip is used for this Hallow's End event, originally without the commit you had 4 equal gossip but 1 broken.